demo/apisix/tmail-apisix-plugin-runner/README.md

```diff
@@ -19,6 +19,27 @@ mvn clean package
       - Runner Library `apache-apisix-java-plugin-runner-0.4.0-bin.tar.gz` (original: https://github.com/apache/apisix-java-plugin-runner)
     - Docker build: `docker build -t linagora/apisix:3.2.0-debian-javaplugin .`
 - Import `tmail-apisix-plugin-runner-{version}.jar` into Apisix container. For detail ([Apache APISIX® Java Plugin Runner](https://apisix.apache.org/docs/java-plugin-runner/how-it-works/))
+    - Eg: `-v target/tmail-apisix-plugin-runner-0.0.1-SNAPSHOT.jar:/usr/local/apisix/token_revoker_plugin.jar`
+- Configuration the plugin in `/usr/local/apisix/conf/config.yaml` of apisix container
+    - Append: 
+```yaml
+ext-plugin:
+  path_for_test: /tmp/runner.sock
+  cmd: ['java', '-jar', '-Xmx1g', '-Xms1g', '/usr/local/apisix/token_revoker_plugin.jar']
+```
+- Use `TokenRevokedFilter` Plugin for special route:
+  - Eg: for `/ping` route
```

Comment:
The ping route is bound to not exist anymore though, see https://github.com/linagora/tmail-backend/pull/673/commits/a789caf3426dd0a33475f741c7a66d7f78f51c2b

Or is that acceptable as an example?

Reply:
IMO acceptable